### ISSUE TITLE
SetAuth missing

### ISSUE THREAD

**Issue by kamyker** [supabase-community/postgrest-csharp#42]
Similar to realtime or js client https://supabase.com/docs/reference/javascript/auth-setauth.

**Comment by acupofjose** [comment]
@kamyker the method you link to references the gotrue auth client - is this what you are speaking of? I don't see that on the existing postgrest-js client.

**Comment by kamyker** [comment]
True, maybe even main repo would be a better place for this issue.

I think there may be a problem with that new method implementation. As a workaround I was doing:
```
await Supabase.Client.InitializeAsync(url, key, new SupabaseOptions()
{
	Headers = new (){{"Authorization", $"Bearer {jwt}"}}
});
```

This didn't work because CurrentSession is null:
```
Supabase.Client.Instance.Auth.CurrentSession.AccessToken = jwt;
```

So I'm guessing this also won't work in some cases https://github.com/supabase-community/gotrue-csharp/blob/4bd2b42dea4778df2331cbdfd2d72d3eb60bd3e5/Gotrue/Client.cs#L644

**Comment by acupofjose** [comment]
Ah! Good Catch, thank you. I've update that on `master` if you'll give it a try for me please!

**Comment by acupofjose** [comment]
I'm also going to transfer this to `gotrue-csharp` for cleanliness.
